src/components/Login.tsx

```diff
@@ -76,15 +89,19 @@ const Login = ({ isPrimary, loginWalletOptions }: LoginProps): JSX.Element => {
       const waddr = await wallet.wallet(selectedType).login();
       await loginWithWalletAddress(waddr, selectedType);
     } catch (error) {
+      console.log("Login error", error);
```

Comment:
If we're trapping it but it's an error, maybe this would be better logged as console.warn.